cmds/boot/systemboot/main.go

```diff
@@ -338,14 +341,23 @@ func main() {
 		log.Print("Falling back to the default boot sequence")
 		for {
 			for _, bootcmd := range defaultBootsequence {
+				if _, err := exec.LookPath(bootcmd[0]); err != nil {
+					log.Printf("No Path: %v", bootcmd)
+					continue
+				}
 				if debugEnabled {
-					bootcmd = append(bootcmd, "-d")
```

Comment:
is there any way to fix the commands to use the same switch (-d or -v)? I guess that can happen later but this is definitely confusing. The are only different because different people wrote them. what do you think? Of course if we change them it may break a script :-(

Reply:
Done. Thank you